### PR TITLE
[English documentation] align the contents of defend.txt with jdefend.txt

### PR DESCRIPTION
**Pull request by backwardsEric**
Make some minor changes for readability.

### CODE REVIEW

File: lib/help/defend.txt [hunk 1]
```diff
@@ -299,69 +414,85 @@ of Speed
...
+Artifacts never break. However, they may deteriorate and, if caught
+in the effect range of *destruction, may disappear forever.
+
+In Hengband, there are two types of artifacts: fixed and random.
+A fixed artifact has a predetermined name and attributes, so,
+if you play the game many times, you may encounter the same fixed
+artifact. The name and attributes of a random artifact are determined
+when it is generated, so it is extremely unlikely that you would see
+that artifact again in another game. A fixed artifact will have a
+"★" symol in its name. A random one will have a "☆" in its name.
```
Comment (sikabane-works) [inline lib/help/defend.txt:449]: English version doesn't add ★ and ☆ symbols to artifact header.

**Review by sikabane-works — COMMENTED** [review]
Thank you for additions and proofreading. Please fix a only difference feature about artifact, and sign your name at footer. :)

File: lib/help/defend.txt (hunk 1 above)
Comment (backwardsEric) [reply lib/help/defend.txt]: I removed the sentences mentioning ★ and ☆ for artifacts.  I'll have to fix up the similar line that was included in attack.txt in a previous pull request.

I added a line to footer.  Should the Japanese help files that have had substantive recent changes be changed as well? The last line in jdefend.txt says it was updated for 1.0.11; jattack.txt says it was updated for 1.5.2.

**Review by sikabane-works — APPROVED** [review]
Thank you. I Approved it.

**Pull request merged**